support-frontend/assets/helpers/abTests/vatAffectedCountries.ts

```diff
@@ -0,0 +1,41 @@
+import type { IsoCountry } from 'helpers/internationalisation/country';
```

Comment:
I created this file so that the `countriesAffectedByVATStatus` list could be kept around without complaining that it was declared but never read

Reply:
Don't we need to use this? Is make-it-annual-nudge even displayed for these countries? Maybe its my lack of full understanding here?